### PR TITLE
Add the possibility to specify a path param name instead of an index

### PR DESCRIPTION
**Pull request by OlivierCuyp**

### CODE REVIEW

File: src/languageLookups/path.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
 export default {
   name: 'path',
 
-  lookup(req, res, options) {
+  lookup (req, res, options) {
     let found;
 
-    if (options.lookupFromPathIndex !== undefined && typeof req !== 'undefined') {
+    if (typeof req !== 'undefined') {
```
Comment (jamuhl) [inline src/languageLookups/path.js]: shouldn't that be if (typeof req === 'undefined') ?

Comment (OlivierCuyp) [inline src/languageLookups/path.js]: Yes, you are correcting now...

**Comment by OlivierCuyp** [comment]
@jamuhl corrected, should be good now. I also build the project (which I initially forgot :'( )

**Comment by jamuhl** [comment]
published v0.2.0

**Comment by jamuhl** [comment]
thanks for adding this...

**Comment by OlivierCuyp** [comment]
@jamuhl Thanks for merging this.
Quick question, why do you keep the build in the repository?
You should ignore it and build on `prepublish` like here: https://github.com/nekuz0r/i18next-smart-bucket-backend/blob/master/package.json#L29

**Comment by jamuhl** [comment]
@OlivierCuyp agree...will change it when time allows it.

https://github.com/i18next/i18next-express-middleware/issues/11

**Pull request merged**